src/main/java/ai/knowly/langtorch/processor/cohere/generate/CohereGenerateProcessorConfig.java

```diff
@@ -0,0 +1,84 @@
+package ai.knowly.langtorch.processor.cohere.generate;
+
+import ai.knowly.langtorch.processor.ProcessorConfig;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Optional;
+
+@AutoValue
+public abstract class CohereGenerateProcessorConfig implements ProcessorConfig {
+  public static Builder builder() {
+    return new AutoValue_CohereGenerateProcessorConfig.Builder()
+        .setModel("command")
```

Comment:
constant?

Reply:
Yes, made the string into a constant now.